tests/conftest.py

```
@@ -0,0 +1,120 @@
+    return orig
+
+
+def run_builder2ibek(xml: Path, out_yaml: Path) -> None:
+    """Convert builder XML to ioc.yaml using the installed builder2ibek.
+
+    builder2ibek is a CI-only dependency (see pyproject [dependency-groups]).
+    Pinning it means Renovate raises a PR -- and runs these tests -- whenever a
+    new builder2ibek is released, catching incompatibilities with rtems-proxy.
+    """
+    out_yaml.parent.mkdir(parents=True, exist_ok=True)
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)